### ISSUE TITLE
[demo_app] no data displayed after launching demo app

### ISSUE THREAD

**Issue by cdmuhs** [hackoregon/transportation-congestion-analysis#36]
Hi I'm trying to launch the demo app per these instructions here: https://github.com/hackoregon/2019-transportation-data-science/tree/master/src/demo_app 

I can get the app going on `localhost:8052` but no data is displayed. Am I missing a step?

![image](https://user-images.githubusercontent.com/9286624/62254767-4f253d00-b3af-11e9-8ddd-f8eb2b3c2969.png)

**Comment by geophpherie** [comment]
Nope you probably did everything right. At one point I edited the app to pull data from the API (but only if you have it running locally). It wasn't an issue when I was the only one using it :). It has some sample data built in, I just need to point it back to that data instead. If you're interested in seeing it in action, it's not a big lift for me to do that, and I should leave it operational anyway.

**Comment by geophpherie** [comment]
@cdmuhs Let me know if you have any other questions or issues! 

I provided some context that may or may not have been confusing ... basically the app first worked off some local `.csv` files, then I configured it to run with the API we are building (but isn't deployed yet), hence the issue of no data. This fix allows it to work with both, although I don't recall at the moment exactly what query created the `.csv` files. Again, it was just a demo to help us plan out our front end, so not everything was fleshed out. Enjoy!
